rootfs/usr/bin/cec-control

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+ACTION="$1"
+
+
+# Source environment from ~/.config/environment.d with user overrides
+set -a
+for i in /home/gamer/.config/environment.d/*.conf ;
```

Comment:
Does the service need to be run as root? If not, it can be a user service and you' ll know for sure the $HOME variable. If not, you can have a configuration file under `/etc` to configure the daemon.

Reply:
Why does the user need to be added to the dialout group? Cold a UDEV rule for the CEC device give the user full permissions without changing the group permissions?

Reply:
Any traction on this? IMO running as a user service seems like the best course, I'm against using hard coded directories for user when environment variables are set.  I'm waiting to test this on my CEC controller once these changes are made.